op-guide/tidb-config-file.md

```diff
@@ -57,7 +57,7 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 ### `slow-query-file`
```

Comment:
we should add an introduction about slow query log differents feature with master or before 2.1.8 on the official document.

Reply:
The user can look into the docs of 2.1, I think no need to mention here.

Reply:
done.